extensions.json

```diff
@@ -999,7 +1049,8 @@
     "repository": "https://github.com/ryu1kn/vscode-edit-with-shell"
   },
   "saltud.type4py": {
-    "repository": "https://github.com/saltudelft/type4py-vscode-ext"
+    "repository": "https://github.com/saltudelft/type4py-vscode-ext",
+    "prepublish": "[ $VERSION = '0.2.0' ] && npm run package-web || true"
```

Comment:
I think we can revert the changes - we can just have it fail until the next version, so that we don't have to remove the version check afterwards.

Reply:
I think it might be a good idea to perform an individual publish, then
What do you think?

Reply:
Sounds good, will do that.

Reply:
Awesome! Thanks for your effort 😄

Reply:
https://github.com/open-vsx/publish-extensions/runs/5821413325?check_suite_focus=true#step:5:4 - published 0.2.0